docs/security.html

```diff
@@ -1266,6 +1264,13 @@ <h4 class="anchor-heading"><a id="security_authz_cli" class="anchor-link"></a><a
             <td></td>
             <td>ResourcePattern</td>
         </tr>
+        <tr>
```

Comment:
Can we also update https://kafka.apache.org/documentation/#security_authz_primitives with newly Operations, Resource and also update [table the here](https://kafka.apache.org/documentation/#operations_resources_and_protocols)

Reply:
Done.